gamemodes/terrortown/entities/weapons/weapon_ttt_binoculars.lua

```diff
@@ -260,23 +272,34 @@ if CLIENT then
 		local client = LocalPlayer()
 		local ent = tData:GetEntity()
 
-		if not IsValid(client) or not client:IsTerror() or not client:Alive() then return end
+		if not IsValid(client) or not client:IsTerror() or not client:Alive() then
+			return
+		end
 
 		local c_wep = client:GetActiveWeapon()
 
-		if not IsValid(c_wep) then return end
+		if not IsValid(c_wep) then
+			return
+		end
 
-		if not IsValid(ent) or ent:GetClass() ~= "prop_ragdoll" or c_wep:GetClass() ~= "weapon_ttt_binoculars" then return end
+		if
+			not IsValid(ent)
+			or ent:GetClass() ~= "prop_ragdoll"
+			or c_wep:GetClass() ~= "weapon_ttt_binoculars"
+		then
+			return
+		end
 
 		-- draw progress
-		if not c_wep:GetProcessing() then return end
-
-		local progress = mathRound(mathClamp((CurTime() - c_wep:GetStartTime()) / c_wep.ProcessingDelay * 100, 0, 100))
+		if not c_wep:GetProcessing() then
+			return
+		end
 
-		tData:AddDescriptionLine(
-			GetPT("binoc_progress", {progress = progress}),
-			hud_color
+		local progress = mathRound(
+			mathClamp((CurTime() - c_wep:GetStartTime()) / c_wep.ProcessingDelay * 100, 0, 100)
 		)
```

Comment:
there is no consistent way of styling this as these are wildly different cases of chaining wildly different functions and their parameters.

In these cases a cutoff point is chosen to stay below 100 characters.

Reply:
Honestly, I see no benefit then. Cutting the lines short by introducing different styles of functions is no improvement imho. We should select a single style and stick to it instead of randomly changing it

Reply:
it is not changed randomly. it is perfectly deterministic.

Reply:
We can argue all day about that. All I'm trying to say is that we should select one style and stick to it. I mean it might be deterministic to have three cases

- try to fit the whole line
- try to split it at the `=`
- try to split it at the `,`

but the second case looks like the only thing that annoys me about wordwrap. It looks so aribitrary to wrap that line there.